packages/objects/src/primitives/FarcasterKey.ts

```diff
@@ -0,0 +1,7 @@
+import { Brand, make } from "ts-brand";
+
+// Fascaster's Register signature
+// https://docs.farcaster.xyz/reference/contracts/reference/id-gateway#register-signature
+
+export type FarcasterKey = Brand<string, "FarcasterKey">;
```

Comment:
Is this a Private key, public key, or a key ID? I'm not sure what a FarcasterKey would actually be.

Reply:
The keys are technically `ED25519PublicKey` keys, replaced with that and removed `FarcasterKey`.